lib/tasks/lint.rake

```diff
@@ -1,4 +1,8 @@
-desc "Run RuboCop"
-task lint: :environment do
-  sh "bundle exec rubocop --format clang"
+unless Rails.env.production?
+  desc "Run RuboCop"
+  task lint: :environment do
+    sh "bundle exec rubocop --format clang"
+  end
+
+  task(:default).prerequisites << task(:lint)
```

Comment:
We're trying to avoid this behaviour, as it surprising when adding something else into the `default` task; the order will depend on the load order of these files.

We could just add `unless Rails.env.production?` [here](https://github.com/alphagov/email-alert-api/pull/1239/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffL10).

Reply:
Yeah, I realised that's a less nice solution. I've amended the commit which basically wraps that in `unless Rails.env.production?`.